source/CardGames.py

```diff
@@ -355,8 +446,21 @@ def winnnerCheck(self, house, player):
       print("It's a push! You both have blackjack.")
       betting_box.collect(blackjackOutcome)
       print(f"You now have ${player_name.money}")
-      
+   
     else:
+    
+#################### DOUBLE DOWN ############################
+      dub_down = input("\nWould you like to double down? (y/n) ")
+      if dub_down == "y":
+        if player_name.money >= int(bet):
```

Comment:
dub_down = input("\nWould you like to double down? (y/n) ") should be put in after if player_name.money >= int(bet): . We should check if they can even double down before we ask them if they want to double down. This way we can get rid of the else statement and avoid cluttering up stdout with excessive print statements.

Reply:
Thats a good point, I hadn't thought of that. Is the state of the player not having enough money the only case that would prevent the player from doubling down?

Reply:
The only time you can double down is after the player's first two cards have been dealt and if they have enough money to double their bet then they can.